Application has thrown an uncaught exception and is terminated:
TypeError: Cannot read property '_id' of undefined
    at Promise.module.exports.Blog.findMostRecent (\\10.207.188.48\volume-22-default\b6e1596fdce18f78aadb\eb3fb43bd98e49bd92d5e543ca052dcc\site\wwwroot\modules\blog.js:97:21)
    at Promise.<anonymous> (\\10.207.188.48\volume-22-default\b6e1596fdce18f78aadb\eb3fb43bd98e49bd92d5e543ca052dcc\site\wwwroot\node_modules\mongoose\lib\promise.js:128:8)
    at Promise.EventEmitter.emit (events.js:88:17)
    at Promise.emit (\\10.207.188.48\volume-22-default\b6e1596fdce18f78aadb\eb3fb43bd98e49bd92d5e543ca052dcc\site\wwwroot\node_modules\mongoose\lib\promise.js:66:38)
    at Promise.error (\\10.207.188.48\volume-22-default\b6e1596fdce18f78aadb\eb3fb43bd98e49bd92d5e543ca052dcc\site\wwwroot\node_modules\mongoose\lib\promise.js:91:15)
    at Query.findOne (\\10.207.188.48\volume-22-default\b6e1596fdce18f78aadb\eb3fb43bd98e49bd92d5e543ca052dcc\site\wwwroot\node_modules\mongoose\lib\query.js:1599:29)
    at exports.tick (\\10.207.188.48\volume-22-default\b6e1596fdce18f78aadb\eb3fb43bd98e49bd92d5e543ca052dcc\site\wwwroot\node_modules\mongoose\lib\utils.js:404:16)
    at Collection.findOne (\\10.207.188.48\volume-22-default\b6e1596fdce18f78aadb\eb3fb43bd98e49bd92d5e543ca052dcc\site\wwwroot\node_modules\mongodb\lib\mongodb\collection.js:1009:28)
    at Cursor.nextObject.commandHandler (\\10.207.188.48\volume-22-default\b6e1596fdce18f78aadb\eb3fb43bd98e49bd92d5e543ca052dcc\site\wwwroot\node_modules\mongodb\lib\mongodb\cursor.js:604:48)
    at Db._executeQueryCommand (\\10.207.188.48\volume-22-default\b6e1596fdce18f78aadb\eb3fb43bd98e49bd92d5e543ca052dcc\site\wwwroot\node_modules\mongodb\lib\mongodb\db.js:1702:5)
